src/lib.rs

```diff
@@ -208,6 +222,18 @@ impl<'a> Ammonia<'a> {
             child.parent.replace(None);
             dom.append(&parent.clone(), NodeOrText::AppendNode(child.clone()));
         }
+        pass
+    }
+
+    fn fix_child(&self, child: &mut Handle, link_rel: &Option<StrTendril>) {
```

Comment:
Mhh since this is so generic, I think you should pass the whole config struct. Otherwise it'll end up `fix_child(&self, thing_for_rel, thing_for_something,...)`.